src/annotation-targets/parse-selector.ts

```diff
@@ -1,13 +1,16 @@
-import { ParsedSelector, SupportedSelectors, TemporalSelector } from './selector-types';
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { ParsedSelector, SupportedSelectors, TemporalSelector, SvgSelector } from './selector-types';
 import { Selector } from '@iiif/presentation-3';
+import { NormalizedSvgPathCommand, parseAndNormalizeSvgPath } from './normalize-svg';
+import { flattenCubicBezier, flattenQuadraticBezier } from './bezier';
 
 const BOX_SELECTOR =
   /&?(xywh=)?(pixel:|percent:|pct:)?([0-9]+(?:\.[0-9]+)?),([0-9]+(?:\.[0-9]+)?),([0-9]+(?:\.[0-9]+)?),([0-9]+(?:\.[0-9]+)?)/;
 
 // Does not support 00:00:00 or 00:00 formats.
 const TEMPORAL_SELECTOR = /&?(t=)(npt:)?([0-9]+(.[0-9]+)?)?(,([0-9]+(.[0-9]+)?))?/;
 
-export function parseSelector(source: Selector | Selector[]): ParsedSelector {
+export function parseSelector(source: Selector | Selector[], domParser?: DOMParser): ParsedSelector {
```

Comment:
```suggestion
export function parseSelector(source: Selector | Selector[], { domParser }:  { domParser: DOMParser } = {}): ParsedSelector {
```
I think it might be good to have more options like this in the future like maybe a: 
```
normaliseSvg(svg: SVGElement): SVGElement;
```